### DISCUSSION TITLE
posts/2019-09-28-secure-hash

### DISCUSSION THREAD

**Discussion by giscus** [sunbjt/blog#10] [Announcements]
# posts/secure-hash

随着我们注册的网站和 App 越来越多，有一个问题一直困扰着我：  我的密码真心不够用！  经常几个可能密码重复的尝试，时不时网站就报超过尝试次数。以及还有一个更为可怕的风险：如果所有的网站如果使用同样的密码，任意一个网站只要发生安全泄露（这几年发生次数不少），那基本你在其他网站就属于裸奔了，其他人可以利用你的统一密码作出一系列你不能想象的行为。 1Password 给我了一些启发，它可以保证你每

https://bjt.name/2019/09/28/secure-hash.html

**Comment by jinghaobi** [comment]
挺好的，简单易用
